CHANGELOG.md

```diff
@@ -1,33 +1,25 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
-## 2.1.16 "TBD" - TBD
-
-<!---
-Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 2, 2020 14:00 -0800
---->
-
-Bug fixes:
-* Fix to make sure we don't expect a valid Docker socket when running Kubernetes monitor in CRI mode. This fixes an issue preventing the K8s monitor from running in CRI mode if Docker is not available.
-
-Misc:
-* The default value for the `k8s_cri_query_filesystem` Kubernetes monitor config option (set via the `SCALYR_K8S_CRI_QUERY_FILESYSTEM` environment var) has changed to `True`. This means that by default when in CRI mode, the monitor will only query the filesystem for the list of active containers, rather than first querying the Kubelet API. If you wish to revert to the original default to prefer using the Kubelet API, set `SCALYR_K8S_CRI_QUERY_FILESYSTEM` the environment variable to "false" for the Scalyr Agent daemonset.
-
 ## 2.1.15 "Endora" - December 2, 2020
 
 <!---
 Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 2, 2020 14:00 -0800
 --->
-
+Feature:
+* The configuration option `default_workers_per_api_key` which allows to establish more than one simultaneous sessions(workers) with the Scalyr servers.
+* The configuration option `use_multiprocess_copying_workers` which allows to start a all sessions(workers) in a separate processes, thereby providing more CPU resources to the agent.
```

Comment:
We also need something like:

"Ability to upload logs to different Scalyr team accounts by specifying different API keys for different log files.  See [RELEASE_NOTES](https://github.com/scalyr/scalyr-agent-2/blob/master/RELEASE_NOTES.md) for more details."

Of course, we need to make that section in the release notes as well.

Reply:
Yes, I'll put the same text as I've written in the scalyr docs. I Just think that it would be better to pass your review first and then just ctrl+c,v it.